### PR TITLE
Add Turkish localization for security chaos engineering

### PR DESCRIPTION
**Pull request by zehrakarakayaa**
### Checklist before opening this PR (put `x` in the checkboxes)
- [ ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/tr/security-chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: Kaos Güvenliği Mühendisliği
+status: Completed
+category: concept
+tags: ["güvenlik", "metodoloji", ""]
+---
+
+Kaos güvenliği mühendisliği (SCE), [kaos mühendisliği](/chaos-engineering/) tabanlı bir disiplindir. SCE, sistemin çalkantılı ve kötü niyetli koşullara dayanma becerisine güven oluşturmak için dağıtılmış bir sistem üzerinde proaktif güvenlik deneyleri gerçekleştirir. Kaos güvenliği mühendisleri, bunu başarmak için kararlı durum, hipotez, sürekli doğrulama, öğrenilen dersler ve hafifletme uygulamasını kapsayan bilimsel yöntem döngülerini kullanır.
+
+## Hangi Sorunları Çözer
+[Site güvenilirliği mühendisleri](/site-reliability-engineering/) (SRE'ler) ve siber güvenlik mühendisleri için ana öncelik, sıfır kesintiye ulaşmak ve iş etkisini en aza indirme hedefiyle hizmeti olabildiğince hızlı bir şekilde geri yüklemektir. SRE'ler ve siber güvenlik mühendisleri hem arıza öncesi hem de arıza sonrası olay durumlarıyla ilgilenir. Çoğu güvenlik sorununun hızlı bir şekilde tespit edilip düzeltilmesi zordur, bu da uygulama veya sistem işlevselliğini etkiler. Ek olarak, güvenlik olaylarının geliştirme aşamasında ortaya çıkarılması genellikle zordur.
```
Comment (aliok) [inline content/tr/security-chaos-engineering.md:11]:
```suggestion
[Site güvenilirliği mühendisleri](./site-reliability-engineering/) (SRE'ler) ve siber güvenlik mühendisleri için ana öncelik, sıfır kesintiye ulaşmak ve iş etkisini en aza indirme hedefiyle hizmeti olabildiğince hızlı bir şekilde geri yüklemektir. SRE'ler ve siber güvenlik mühendisleri hem arıza öncesi hem de arıza sonrası olay durumlarıyla ilgilenir. Çoğu güvenlik sorununun hızlı bir şekilde tespit edilip düzeltilmesi zordur, bu da uygulama veya sistem işlevselliğini etkiler. Ek olarak, güvenlik olaylarının geliştirme aşamasında ortaya çıkarılması genellikle zordur.
```

File: content/tr/security-chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Kaos Güvenliği Mühendisliği
+status: Completed
+category: concept
+tags: ["güvenlik", "metodoloji", ""]
+---
+
+Kaos güvenliği mühendisliği (SCE), [kaos mühendisliği](/chaos-engineering/) tabanlı bir disiplindir. SCE, sistemin çalkantılı ve kötü niyetli koşullara dayanma becerisine güven oluşturmak için dağıtılmış bir sistem üzerinde proaktif güvenlik deneyleri gerçekleştirir. Kaos güvenliği mühendisleri, bunu başarmak için kararlı durum, hipotez, sürekli doğrulama, öğrenilen dersler ve hafifletme uygulamasını kapsayan bilimsel yöntem döngülerini kullanır.
```
Comment (aliok) [inline content/tr/security-chaos-engineering.md:8]:
```suggestion
Kaos güvenliği mühendisliği (SCE), kaos mühendisliği tabanlı bir disiplindir. SCE, sistemin çalkantılı ve kötü niyetli koşullara dayanma becerisine güven oluşturmak için dağıtılmış bir sistem üzerinde proaktif güvenlik deneyleri gerçekleştirir. Kaos güvenliği mühendisleri, bunu başarmak için kararlı durum, hipotez, sürekli doğrulama, öğrenilen dersler ve hafifletme uygulamasını kapsayan bilimsel yöntem döngülerini kullanır.
```

We don't have the linked page yet.

File: content/tr/security-chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Hangi Sorunları Çözer
+[Site güvenilirliği mühendisleri](/site-reliability-engineering/) (SRE'ler) ve siber güvenlik mühendisleri için ana öncelik, sıfır kesintiye ulaşmak ve iş etkisini en aza indirme hedefiyle hizmeti olabildiğince hızlı bir şekilde geri yüklemektir. SRE'ler ve siber güvenlik mühendisleri hem arıza öncesi hem de arıza sonrası olay durumlarıyla ilgilenir. Çoğu güvenlik sorununun hızlı bir şekilde tespit edilip düzeltilmesi zordur, bu da uygulama veya sistem işlevselliğini etkiler. Ek olarak, güvenlik olaylarının geliştirme aşamasında ortaya çıkarılması genellikle zordur.
+
+## Nasıl Yardımcı Olur
+Kaos güvenliği mühendisliği, [gözlemlenebilirlik](/observability/) ve siber dayanıklılık uygulamaları üzerine kurulmuştur. "Bilinmeyen bilinmeyenleri" ortaya çıkarmayı ve sistemde güven oluşturmayı, siber dayanıklılığı artırmayı ve gözlemlenebilirliği iyileştirmeyi amaçlamaktadır.
```
Comment (aliok) [inline content/tr/security-chaos-engineering.md:14]:
```suggestion
Kaos güvenliği mühendisliği, [gözlemlenebilirlik](./observability/) ve siber dayanıklılık uygulamaları üzerine kurulmuştur. "Bilinmeyen bilinmeyenleri" ortaya çıkarmayı ve sistemde güven oluşturmayı, siber dayanıklılığı artırmayı ve gözlemlenebilirliği iyileştirmeyi amaçlamaktadır. 
```

File: content/tr/security-chaos-engineering.md (hunk 1 above)
Comment (aliok) [reply content/tr/security-chaos-engineering.md:11]: Let's link to the Turkish page here.

File: content/tr/security-chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Nasıl Yardımcı Olur
+Kaos güvenliği mühendisliği, [gözlemlenebilirlik](/observability/) ve siber dayanıklılık uygulamaları üzerine kurulmuştur. "Bilinmeyen bilinmeyenleri" ortaya çıkarmayı ve sistemde güven oluşturmayı, siber dayanıklılığı artırmayı ve gözlemlenebilirliği iyileştirmeyi amaçlamaktadır. 
+Mühendislik ekiplerinin, karmaşık altyapı, platformlar ve dağıtılmış sistemlerdeki güvenlik sorunlarına ilişkin anlayışı giderek gelişecektir. SCE, tüm ürünün siber dayanıklılığını artırır, gizli güvenlik sorunlarını ortaya çıkarır, klasik kör noktaları ortaya çıkarır ve ekipleri kritik uç durumlara hazırlar. Bu yaklaşım, [SRE](/site-reliability-engineering/)'lerin, [DevOps](/devops/)'un ve [DevSecOps](/devsecops/) mühendislerinin sistemde güven oluşturmasına, siber dayanıklılığı artırmasına ve gözlemlenebilirliği iyileştirmesine yardımcı olur.
```
Comment (aliok) [inline content/tr/security-chaos-engineering.md:15]:
```suggestion
Mühendislik ekiplerinin, karmaşık altyapı, platformlar ve dağıtılmış sistemlerdeki güvenlik sorunlarına ilişkin anlayışı giderek gelişecektir. SCE, tüm ürünün siber dayanıklılığını artırır, gizli güvenlik sorunlarını ortaya çıkarır, klasik kör noktaları ortaya çıkarır ve ekipleri kritik uç durumlara hazırlar. Bu yaklaşım, [SRE](./site-reliability-engineering/)'lerin, [DevOps](./devops/)'un ve DevSecOps mühendislerinin sistemde güven oluşturmasına, siber dayanıklılığı artırmasına ve gözlemlenebilirliği iyileştirmesine yardımcı olur.
```

Similar to above, some pages don't exist and some links are to the English ones.

**Review by aliok — APPROVED** [review]

**Review by eminalemdar — APPROVED** [review]

**Comment by aliok** [comment]
@zehrakarakayaa this was opened to the targeting a wrong branch (`main`) and I now changed the base to `dev-tr` branch.

However, it needs a rebase.

**Review by symys — APPROVED** [review]

**Comment by iamNoah1** [comment]
@symys @aliok I am not sure if rebasing will help. IMHO the PR should be closed and a new one opened where the changes are being made based on the correct branch. The commit could be cherry-picked still.

**Pull request closed without merging**